core/src/main/java/com/netflix/conductor/service/MetadataServiceImpl.java

```diff
@@ -119,11 +122,22 @@ public void updateWorkflowDef(WorkflowDef workflowDef) {
     /**
      * @param workflowDefList Workflow definitions to be updated.
      */
-    public void updateWorkflowDef(List<WorkflowDef> workflowDefList) {
+    public BulkResponse updateWorkflowDef(List<WorkflowDef> workflowDefList) {
+        BulkResponse bulkResponse = new BulkResponse();
         for (WorkflowDef workflowDef : workflowDefList) {
-            workflowDef.setUpdateTime(System.currentTimeMillis());
-            metadataDAO.updateWorkflowDef(workflowDef);
+            try {
+                updateWorkflowDef(workflowDef);
+                bulkResponse.appendSuccessResponse(workflowDef.getName());
+            } catch (Exception e) {
+                LOGGER.error(
+                        "bulk updateWorkflowDef exception, workflowDef name {}, message: {} ",
```

Comment:
```suggestion
                        "Bulk update workflow def failed, name {}",
```
message is redundant since the logger prints the message and the stacktrace by default.